doc/api/crypto.md

```diff
@@ -1365,6 +1365,8 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [Buffers][`Buffer`], `TypedArray`, or `DataView`s. If the cipher does not need
 an initialization vector, `iv` may be `null`.
 
+An initialization vector should be unpredictable; ideally, they will be cryptographically random. They do not have to be secret: IVs are typically just added to ciphertext messages in plaintext. It may sound contradictory that something has to be unpredictable, but doesn’t have to be secret; it’s important to remember that an attacker must not be able to predict ahead of time what a given IV will be.
```

Comment:
Lines need to be wrapped at 80 characters.

Reply:
aren't they? strange. I'll do that promptly, sorry.

Reply:
oh, wait. It's just one single huge line. wtf.